perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -117,9 +117,9 @@ sub short_name {
 }
 
 sub find_closest {
-    my ( $self, $problem, $as_data ) = @_;
+    my ($self, $problem) = @_;
 
-    my $data = $self->SUPER::find_closest($problem, $as_data);
+    my $closest = $self->SUPER::find_closest($problem);
 
     my $mapit_url = FixMyStreet->config('MAPIT_URL');
     my ($lat, $lon) = map { Utils::truncate_coordinate($_) } $problem->latitude, $problem->longitude;
```

Comment:
This line is now crashing with `Can't call method "latitude" on unblessed reference` because `$problem` is a hashref when called from `Around->location_closest_address`